modules/labs-react/search-form/stories/SearchForm.stories.mdx

```diff
@@ -0,0 +1,61 @@
+import {Meta} from '@storybook/addon-docs/blocks';
+import {SearchForm} from '@workday/canvas-kit-labs-react/search-form';
+// examples
+import {Basic} from './examples/Basic';
+import {Growth} from './examples/Growth';
+import {RTL} from './examples/RTL';
+import {Theming} from './examples/Theming';
+import {ThemeAttributes} from './examples/PropTables.splitProps';
+
+<Meta title="Labs/Search Form" component={SearchForm} />
```

Comment:
Most of our components have a React folder at the end of the Storybook path even if there isn't a CSS version of the component, though I noticed `Box`, `Flex`, and `Stack` forgo this convention.

I don't have a strong opinion here, just pointing it out. It's more noticeable with `SearchForm` since the reduced depth of its story paths results in a different icon in the side nav.

<img width="234" alt="sb-search-form" src="https://user-images.githubusercontent.com/146020/134999633-ec44ab19-7747-4ddf-aa48-fc9fa0521953.png">